openfisca_us/variables/gov/states/md/tax/income/credits/md_cdcc.py

```diff
@@ -0,0 +1,37 @@
+from openfisca_us.model_api import *
+import numpy as np
+
+
+class md_cdcc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MD CDCC"
+    documentation = "Maryland Child and Dependent Care Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://casetext.com/statute/code-of-maryland/article-tax-general/title-10-income-tax/subtitle-7-income-tax-credits/section-10-716-for-child-care-or-dependent-care"
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
+        agi = tax_unit("adjusted_gross_income", period)
+        p = parameters(period).gov.states.md.tax.income.credits.cdcc
+        # Eligibility is based on AGI.
+        in_md = tax_unit.household("state_code_str", period) == "MD"
```

Comment:
Why do we need to include the residency check here? Because of federal AGI is the input?

Reply:
Passing a MD AGI variable would still give >0 md_cdcc to people outside MD, since it includes zero-earners. We're not consistent around this yet (Nikhil's work on a filter will make it consistent) but I did it here so I could estimate the budgetary impact without zeroing out non-MD household weights.